cmip6-downscaling/conda-envs/analysis.yaml

```
@@ -43,6 +45,7 @@ dependencies:
   - rasterio
   - rechunker=0.3.3
```

@orianac / @norlandrhagen - should we update this to 0.4.2?
```suggestion
  - rechunker=0.4.2
```